tetris-PRG.info

```diff
@@ -816,7 +816,15 @@ LABEL { ADDR $9B96; NAME "@checkForBorrow"; };
 LABEL { ADDR $9BA6; NAME "@gameTypeA"; };
 LABEL { ADDR $9BA8; NAME "incrementLines"; };
 LABEL { ADDR $9BFE; NAME "addHoldDownPoints"; };
+LABEL { ADDR $9C18; NAME "@noLowOverflow"; };
+LABEL { ADDR $9C27; NAME "@noHighOverflow"; };
 LABEL { ADDR $9C2D; NAME "addLineClearPoints"; };
```

Comment:
FYI or future: it would be good/fine/appropriate to change this to an `@` label as well.